kanbananza-backend/src/main/DTO/card.js

```diff
@@ -0,0 +1,15 @@
+export default class CardDTO {
+  constructor({ id, label, columnId }) {
+    this.id = id;
+    this.label = label;
+    this.columnId = columnId;
+  }
+
+  static fromDocument(card) {
+    return new this({
+      id: card._id,
+      label: card.label,
+      column: card.columnId,
+    });
+  }
+}
```

Comment:
we moved this to the model classes, check the latest code

Reply:
Where is the latest code? not on main?

Reply:
oh nvm